From c18ef79f58a472aa80522e61b6ece1f384444a29 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:13:14 -0400
Subject: [powerpc] powernv: Reduce panic timeout from 180s to 10s

Message-id: <1410545655-205645-566-git-send-email-dzickus@redhat.com>
Patchwork-id: 94515
O-Subject: [RHEL7 PATCH 565/626] powerpc/powernv: Reduce panic timeout from 180s to 10s
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 4817fc323dad9f08f2cb1aca22e6dc7f894c95d5
Author: Anton Blanchard <anton@samba.org>
Date:   Thu May 1 07:20:04 2014 +1000

    powerpc/powernv: Reduce panic timeout from 180s to 10s

    We've already dropped the default pseries timeout to 10s, do
    the same for powernv.

    Signed-off-by: Anton Blanchard <anton@samba.org>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/platforms/powernv/setup.c b/arch/powerpc/platforms/powernv/setup.c
index 678573c..d9b88fa 100644
--- a/arch/powerpc/platforms/powernv/setup.c
+++ b/arch/powerpc/platforms/powernv/setup.c
@@ -41,6 +41,8 @@
 
 static void __init pnv_setup_arch(void)
 {
+ set_arch_panic_timeout(10, ARCH_PANIC_TIMEOUT);
+
  /* Initialize SMP */
  pnv_smp_init();
 
-- 
1.7.1